srcCpp/perftest_publisher.cxx

```diff
@@ -859,6 +860,12 @@ void perftest_cpp::PrintConfiguration()
 {
     std::ostringstream stringStream;
 
+  #ifdef RTI_CUSTOM_TYPE
+    stringStream << "\nUsing CustomType with the type <"
```

Comment:
```
Using user provided type: "Name"
```

Reply:
Other options:

`Using CustomTypes with the type <XXX>`
`Using custom type: <XXX>`
`Using custom type <XXX>, provided by user`

Reply:
Done